table/index_builder.cc

```diff
@@ -49,4 +51,100 @@ IndexBuilder* IndexBuilder::CreateIndexBuilder(
   assert(false);
   return nullptr;
 }
-}  // namespace rocksdb
+
+PartitionedIndexBuilder* PartitionedIndexBuilder::CreateIndexBuilder(
+    const InternalKeyComparator* comparator,
+    const SliceTransform* prefix_extractor, int index_block_restart_interval,
+    uint64_t index_per_partition, const BlockBasedTableOptions& table_opt) {
+  class DummyFilterBitsBuilder : public FilterBitsBuilder {
+   public:
+    virtual void AddKey(const Slice& key) { assert(0); }
+    virtual Slice Finish(std::unique_ptr<const char[]>* buf) { assert(0); }
+  };
+  FilterBitsBuilder* filter_bits_builder =
```

Comment:
good catch. it was needed when index and filter builders were mixed. but not anymore.